### PR TITLE
Emoji plot

### PR DESCRIPTION
**Pull request by lsomm**
Hi, this is the PR to issue #61

### CODE REVIEW

File: chatminer/visualizations.py [hunk 1]
```diff
@@ -387,3 +388,20 @@ def is_radar_registered():
...
+    df,
+    n_largest=10
+):
+    emoji_list=[]
+    for i in range(0,len(df.index)):
+        get_emojis(df["message"].iloc[i],emoji_list)
+    emoji_list = pd.Series(list(emoji_list))
+    freqs = emoji_list.value_counts()
+    freqs=freqs.nlargest(n=n_largest)
+    return freqs.plot.pie()
```
Comment (joweich) [inline chatminer/visualizations.py:407]: The other visualizations provide the option to hand over the matplotlib axis as a parameter (`ax=None`) and fallback to a default axis via
```
if ax is None:
        _, ax = plt.subplots()
```
I would see this for the piechart as well to be well aligned to the other plots.

**Review by joweich — CHANGES_REQUESTED** [review]
@lsomm many thanks and congratulations to your first PR!
Added some comments that mainly concern code cleaning and code style. 
Would definitely recommend using code linters (pylint, flake8, ...) and an automatic code formatter (black).
Let me know if I can support you with further guidance! :+1:

**Comment by lsomm** [comment]
Thank you so much for the comments! I'll look to it after my exams :-)

**Comment by joweich** [comment]
@lsomm are you working on this? :slightly_smiling_face:

**Comment by lsomm** [comment]
Hi, sorry for staying under the radar for so long. Long story short: I might, not now though. If you want to close the PR, thats fine for me.

**Comment by joweich** [comment]
> Hi, sorry for staying under the radar for so long. Long story short: I might, not now though. If you want to close the PR, thats fine for me. 

@lsomm got you! I will close this for now but happy to reopen it once you get the chance to finish it!
Btw, we have several other open issues if you want to work on something else 🙂

**Pull request closed without merging**